pipelines/defaults.json

```diff
@@ -90,6 +90,16 @@
         "defaultDynamicParas": {
             "testLists"      : ["sanity.functional", "extended.functional", "sanity.system", "extended.system", "special.system", "sanity.jck", "extended.jck", "sanity.openjdk", "dev.system"],
             "numMachines"    : ["3", "3", "3", "3", "5", "4", "8", "2", "5"]
+        },
+        "ebcEnabledTargets":{
+            "linux_x64" :       [""],
+            "aix_ppc64" :       [""],
+            "linux_aarch64" :   [""],
+            "linux_ppc64le" :   [""],
+            "linux_s390x" :     [""],
+            "mac_aarch64" :     [""],
+            "mac_x64" :         [""],
+            "windows-x64" :     [""]
```

Comment:
Do we need to add win-32 here for java8? Possibly risc-v as well (jdk11)? Although we don't have any testing on riscv.